docs/topics/whatsnew-eap.md

```diff
@@ -9,1615 +9,90 @@ _[Released: %kotlinEapReleaseDate%](eap.md#build-details)_
-
-> The K2 Kotlin mode is in Alpha. The performance and stability of code highlighting and code completion have been significantly improved,
-> but not all IDE features are supported yet.
->
-{type="warning"}
-
-Starting from 2024.1, IntelliJ IDEA can use the new K2 compiler to analyze your code with its K2 Kotlin mode. To try it out,
-go to **Settings** | **Languages & Frameworks** | **Kotlin** and select the **Enable the K2-based Kotlin plugin** option.
-
-Learn more about the K2 Kotlin mode in
-[our blog](https://blog.jetbrains.com/idea/2024/03/k2-kotlin-mode-alpha-in-intellij-idea/).
-
-We are actively collecting feedback about K2 Kotlin mode.
-Please share your thoughts in our [public Slack channel](https://kotlinlang.slack.com/archives/C0B8H786P)!
-
-### Leave your feedback on the new K2 compiler
-
-We would appreciate any feedback you may have!
-
-* Provide your feedback directly to K2 developers on Kotlin
-  Slack – [get an invite](https://surveys.jetbrains.com/s3/kotlin-slack-sign-up?_gl=1*ju6cbn*_ga*MTA3MTk5NDkzMC4xNjQ2MDY3MDU4*_ga_9J976DJZ68*MTY1ODMzNzA3OS4xMDAuMS4xNjU4MzQwODEwLjYw)
-  and join the [#k2-early-adopters](https://kotlinlang.slack.com/archives/C03PK0PE257) channel.
-* Report any problems you face with the new K2 compiler
-  in [our issue tracker](https://kotl.in/issue).
-* [Enable the Send usage statistics option](https://www.jetbrains.com/help/idea/settings-usage-statistics.html) to
-  allow JetBrains to collect anonymous data about K2 usage.
-
-## Kotlin/JVM
-
-This version brings the following changes:
+For the latest information about our migration plan,
+see the corresponding issue in [YouTrack](https://youtrack.jetbrains.com/issue/KT-11914).
 
-* [Generate lambda functions using `invokedynamic`](#generate-lambda-functions-using-invokedynamic)
-* [The `kotlinx-metadata-jvm` library is now Stable](#the-kotlinx-metadata-jvm-library-is-now-stable)
+To give you more control over this behavior, in Kotlin %kotlinEapVersion% we’ve introduced two annotations:
+`@ConsistentCopyVisibility` to opt in to the behavior now before we make it the default in a later release.
+`@ExposedCopyVisibility` to opt out of the behavior and suppress warnings at the declaration site. 
+Note that even with this annotation, the compiler still reports warnings when the `.copy()` function is called.
 
-### Generate lambda functions using invokedynamic
+If you want to opt in to the new behavior already in %kotlinEapVersion% for a whole module rather than in individual classes,
+you can use the `-Xconsistent-data-class-copy-visibility` compiler option.
+This option has the same effect as adding the '@ConsistentCopyVisibility' annotation to all data classes in a module.
 
-Kotlin %kotlinEapVersion% introduces a new default method for generating lambda functions using `invokedynamic`.
-This change reduces the binary sizes of applications compared to the traditional anonymous class generation.
+## Kotlin/Native: support for bitcode embedding removed
```

Comment:
```suggestion
## Kotlin/Native: Support for bitcode embedding removed
```

Reply:
Not sure if a capital letter should follow the colon, but for consistency with other headers

Reply:
Thank you, updated the PR